docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -0,0 +1,188 @@
+         
+                 ... other settings
+                 
+                 platform:
+                   openstack:
+                     computeFlavor: <vmComputeFlavorForMasters>
+
+
+- Run the openshift installer:
+
+      ./openshift-install create cluster --log-level debug
+        
+- Wait for installer to complete. 
+
+    If the installer times out waiting for the baremetal workers to complete booting,
+    restart the installation usin the appropriate *wait-for* command:
+    
+        ./openshit-install watier-for bootstrap-complete --log-level debug
+
+## Deploying cluster with BM workers on preexisting network
+Initial cluster is deployed using bare-metal workers only. Bare-metal workers are attached to
+a preexisting network. 
+
+- Requirements: 
+    - An OpenStack subnet which supports attaching vms and baremental servers to is preprovisioned by the user. 
+    
+- Create install-config.yaml:
+    - Set compute.[worker].platform.openstack.type to the baremetal server flavor.
+    - Set control-plane.platform.openstack.type to the vm flavor which will be used by the control-plane masters.
+    - Set platform.openstack.computeFlavor to the vm flavor which will be used by the control-plane masters.
+    - Set platform.openstack.machinesSubnet to the UUID of the preprovisioned subnet. 
+
+        For example:
+ 
+                controlPlane:
+                   platform:
+                     openstack:
+                       type: <vmComputeFlavorForMasters>
+                 
+                 ... other settings
+                 
+                 compute:
+                 - architecture: amd64
+                   hyperthreading: Enabled
+                   name: worker
+                   platform:
+                     openstack:
+                       type: <baremetalComputeFlavor>
+                   replicas: 3
+         
+                 ... other settings
+                 
+                 platform:
+                   openstack:
+                     computeFlavor: <vmComputeFlavorForMasters>
+                     machinesSubnet: <uuidOfPreprovisionedSubnet>
+
+- Run the openshift installer:
+
+        ./openshift-install create cluster --log-level debug
+        
+- Wait for installer to complete. 
+
+    If the installer times out waiting for the baremetal workers to complete booting,
+    restart the installation usin the appropriate *wait-for* command:
+    
+        ./openshit-install watier-for bootstrap-complete --log-level debug
+
+## Deploy cluster with vm workers only and add BM workers after 
+Initial cluster deployment is done with only 
+vm workers attached to the installer-provisioned network. The bare-metal
+workers are added after initial installation. bare-metal workers are
+attached to a preexisting network.
+
+- Requirements: 
+    - Cloud provideer is configure to allow creation of  OpenStack subnets that supports attaching vms and baremental servers to it. 
```

Comment:
Is this a copy-paste error? I read this requirement as "OpenStack lets you add both VMs and BMs to a subnet", but this case is about separate VMs in a tenant network and BMs in a provider network if I'm not mistaken

Reply:
My misunderstanding, baremetals should be in their own pre-existing subnet.  Although this could be a problem when it comes to security group rules. Does the installer apply the necessary security group rules to any extra subnets?

Reply:
The installer does nothing to establish connectivity between two different subnets AFAIK.

In the context of "multiple subnets", establishing connectivity between the nodes is a responsibility we leave to the user.

Reply:
I will correct description and add blurb commenting on connectivity between nodes